docx/oxml/section.py

```diff
@@ -42,12 +44,16 @@ class CT_SectPr(BaseOxmlElement):
     ``<w:sectPr>`` element, the container element for section properties.
     """
     __child_sequence__ = (
-        'w:footnotePr', 'w:endnotePr', 'w:type', 'w:pgSz', 'w:pgMar',
-        'w:paperSrc', 'w:pgBorders', 'w:lnNumType', 'w:pgNumType', 'w:cols',
-        'w:formProt', 'w:vAlign', 'w:noEndnote', 'w:titlePg',
+        'w:headerReference', 'w:footnotePr', 'w:endnotePr', 'w:type', 'w:pgSz',
+        'w:pgMar', 'w:paperSrc', 'w:pgBorders', 'w:lnNumType', 'w:pgNumType',
+        'w:cols', 'w:formProt', 'w:vAlign', 'w:noEndnote', 'w:titlePg',
         'w:textDirection', 'w:bidi', 'w:rtlGutter', 'w:docGrid',
         'w:printerSettings', 'w:sectPrChange',
     )
```

Comment:
I added headerReference to the beginning here because that's where it appears in the analysis